src/http.ts

```diff
@@ -28,8 +28,15 @@ export namespace Http {
                 operationContext: unknown;
             };
         };
+        version: string;
```

Comment:
This is not appropriate place for these fields, since `http.Event` describes regular direct request to cloud function. These fields structure should be described as new type in `apigateway` folder like it is done for authorizer requests (https://github.com/yandex-cloud/function-ts-types/blob/master/src/api-gateway/authorizer.ts). I would suggest to create new module `http` inside and declare types new module with namespace `Http` and types `Event` and `Context` there. Those type should extend regular `Event` and `Context` of course.